src/Sulu/Bundle/ContentBundle/Resources/public/js/validation/types/resourceLocator.js

```diff
@@ -40,7 +40,7 @@ define([
                     var val = this.getValue(),
                         part = App.dom.data($el, 'part');
 
-                    return part.length > 0 && val !== '/' && validator.exec(val);
+                    return part.length > 0 && val !== '/' && validator.test(val);
```

Comment:
why it shouldn't be enough? we said that we will extend this with an overlay and more action in the future. with this no invalid resource-segment can be saved.